concept/thing/RelationImpl.java

```diff
@@ -96,12 +96,12 @@ public RelationTypeImpl getType() {
 
         @Override
         public void addPlayer(RoleType roleType, Thing player) {
-            execute(addPlayerReq(getIID(), protoRoleTypes(roleType), protoThing(player.getIID())));
+            execute(addPlayerReq(getIID(), protoRoleType(roleType), protoThing(player.getIID())));
         }
 
         @Override
         public void removePlayer(RoleType roleType, Thing player) {
-            execute(removePlayerReq(getIID(), protoRoleTypes(roleType), protoThing(player.getIID())));
```

Comment:
This method was taking in a single RoleType, but was confusingly named `protoRoleTypes` with an S